README.md

```diff
@@ -61,34 +61,19 @@ The full set of known keys can be found in the
 `org/edx/mobile/util/Config.java` file or see [additional documentation](<https://openedx.atlassian.net/wiki/display/MA/App+Configuration+Flags>).
 
 
-Flavors
---------------------
-
-There are 2 Flavors in this project:
-
-- **dev**
-    - Disables multidex to drastically speed up build: https://developer.android.com/tools/building/multidex.html#dev-build
-    - Only works for devices with Lollipop and above.
-- **prod**
-    - Used for release builds or devices with OS below Android 5.0 (Lollipop).
-    - Works on all devices we support i.e. Android 4.0 (Ice-cream Sandwich) and above.
-
 Build Variants
 --------------------
 
-There are 6 Build Variants in this project:
+There are 3 Build Variants in this project:
 
-- **devDebug**: Uses dev flavor for debug builds.
-- **devDebuggable**: Uses dev flavor for debug builds with debugging enabled.
-- **devRelease**: Uses dev flavor for release builds that'll work on devices with Android 5.0 (Lollipop) and above.
 - **prodDebug**: Uses prod flavor for debug builds.
 - **prodDebuggable**: Uses prod flavor for debug builds with debugging enabled.
-- **prodRelease**: Uses prod flavor for release builds that'll work on devices with Android 4.0 (Ice-cream Sandwich) and above.
+- **prodRelease**: Uses prod flavor for release builds that'll work on devices with Android 4.1.x (Jelly Bean) and above.
 
 Building For Release
 --------------------
-To build an APK for release, you must specify an application ID and signing key. Create or edit the `gradle.properties` file inside the `OpenEdXMobile` directory of `edx-app-android`. For example:
-
+To build an APK for release, you must specify an application ID and signing key. 
```

Comment:
you _may_ specify an application ID ...

It's also worth mentioning here that the `APPLICATION_ID` key is optional and if it is not present the package identifier is used as the `APPLICATION_ID`.

Reply:
Also, the line you added instead of:
> Create or edit the `gradle.properties` file inside the `OpenEdXMobile` directory of `edx-app-android`. For example:

doesn't make any sense, both are totally different things. The line above needs to be brought back.